website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Deploying a Java OpenLiberty application with PostgreSQL
+sidebar_position: 1
+---
+
+This scenario illustrates deploying a Java application with odo and linking it to an in-cluster PostgreSQL service in the minikube environment.
+
+In this example there are two roles:
+
+1. Cluster Admin - Prepare the cluster by installing the required operators into the cluster
+2. Application Developer - Imports a Java application, creates a Database instance, and connect the application to the Database instance.
+
+## Cluster admin
+---
+
+This section assumes that you have installed minikube and configured it. See [Getting Started > Cluster Setup > Kubernetes](../getting-started/cluster-setup/kubernetes.md).
+
+The cluster admin must install two Operators into the cluster:
+
+1. Operator Backed Service
+2. Service Binding Operator
+
+An _Operator Backed Service_ is an operator that helps in deploying instances of a given service, for example PostgreSQL, MySQL, Redis.
```

Comment:
This statement needs to change once the point 1 above changes. It will explain why the Postgres Operator is required.